refs/biblio.json

```diff
@@ -281,6 +281,90 @@
         "href": "https://standards.cta.tech/kwspub/published_docs/ANSI-CTA-708-E-Preview.pdf",
         "publisher": "CEA"
     },
+    "CEDDL": {
+        "authors": [
```

Comment:
Wow. Um… that _a lot_ of people. Anyway you can shorten that to maybe the main editor(s) and add set the `"etAl"` property to `true`?